src/StatsBase.jl

```diff
@@ -212,7 +212,11 @@ export
     standardize,
     AbstractDataTransform, # the type to represent a abstract data transformation
     ZScoreTransform,       # the type to represent a z-score data transformation
-    UnitRangeTransform     # the type to represent a 0-1 data transformation
+    UnitRangeTransform,    # the type to represent a 0-1 data transformation
+
+    # Reliability
+    Reliability,           # the type to represent reliability scores
+    alpha                  # function to compute Crombach's alpha
```

Comment:
I would prefer some longer name as `alpha` seems too common

Reply:
changed to `crombach_alpha`